inference_cli/main.py

```diff
@@ -14,6 +14,26 @@
 app.add_typer(benchmark_app, name="benchmark")
 
 
+def version_callback(value: bool):
+    if value:
+        from importlib.metadata import version, PackageNotFoundError
+
+        try:
+            inference_version = version('inference')
+            print(f"Inference Version: {inference_version}")
+        except PackageNotFoundError:
+            print(f"Inference Version: using local install")
```

Comment:
Would be great if we can distinguish between package not installed and package installed in editable mode, `importlib.metadata.distribution("inference")` would result in `ModuleNotFoundError` if package was not installed.

Reply:
In my testing I'm only ever getting `PackageNotFoundError`. I agree this distinction would help with crafting the version message.